.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,55 @@
 <!--
 Thanks for sending a pull request!
 
-Please make sure you've read our contributing guidelines and how to submit a pull request (https://github.com/containers/podman/blob/master/CONTRIBUTING.md#submitting-pull-requests).
+Please make sure you've read our contributing guidelines and how to submit a pull request (https://github.com/containers/podman/blob/main/CONTRIBUTING.md#submitting-pull-requests).
 
 In case you're only changing docs, make sure to prefix the pull-request title with "[CI:DOCS]".  That will prevent functional tests from running and save time and energy.
+
+Finally, be sure to sign your commits (e.g. `git commit -s`)
+-->
+
+#### What this PR does / why we need it:
+
+<!---
+Please put your overall PR description here
+-->
+
+#### How to verify it
+
+<!---
+Please specify the precise conditions and/or the specific test(s) which must pass.
+-->
+
+#### Which issue(s) this PR fixes:
+
+<!--
+Please uncomment this block and include only one of the following on a
+line by itself:
```

Comment:
Please also mention to include this in the commit message.

Reply:
You know actually...This might be another item like Tom suggested, we can check for it early on with CI:

If any of the commits, PR title, or this section mention 'Fixes #whatever`: Immediately fail the build if that same thing isn't present in the other expected places.